cocos/network/HttpClient-android.cpp

```diff
@@ -715,7 +713,10 @@ void HttpClient::processResponse(HttpResponse* response, char* responseMessage)
     }
     free(contentInfo);
     
-    strcpy(responseMessage, urlConnection.getResponseMessage());
+    char *messageInfo = urlConnection.getResponseMessage();
+    strcpy(responseMessage, messageInfo);
```

Comment:
This line will trigger an error. I think `responseMessage = messageInfo;` will be better.